packages/mobile/src/utils/time.ts

```diff
@@ -341,3 +342,12 @@ function quickFormat(timestamp: number, i18next: i18nType, formatRule: string) {
     locale: locales[i18next?.language]?.dateFns ?? locales['en-US']?.dateFns,
   })
 }
+
+function toUtcDate(date: Date) {
+  return new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000)
+}
+
+export function getNextMondayAt8pmUtc() {
+  const mondayDate = startOfWeek(Date.now(), { weekStartsOn: 1 })
+  return toUtcDate(mondayDate).getTime() + 20 * ONE_HOUR_IN_MILLIS + 7 * ONE_DAY_IN_MILLIS
+}
```

Comment:
This means we're keeping weekly rewards?
I thought we were switching to monthly rewards after March 1st.

Reply:
Yes, we decided to keep weekly rewards so people keep receiving notifications about the program every week and avoid confusion about needing the average balance across a whole month. We will just reduce rewards to be 1/4th of current ones.